technical-setup.md

```diff
@@ -0,0 +1,60 @@
+# Starport Technical Setup 
+
+To ensure you have a successful experience developing with Starport, ensure that your local system meets these technical requirements.
+
+Starport is supported for the following operating systems:
+
+- GNU/Linux
+- macOS
+- Windows Subsystem for Linux (WSL)
+
+## Install Go
+
+This installation method removes existing Go installations, installs Go in `/usr/local/go/bin/go`, and sets the environment variables.
+
+1. Install [Go](https://golang.org/doc/install) (**version 1.16** or higher).
+
+2. Download the binary release that is suitable for your system.
+
+3. Follow the installation instructions.
+
+**Note:** We recommend not using brew to install Go.
```

Comment:
```suggestion
**Note:** We recommend not using brew to install Go.
```

Not sure why this is the case?

Reply:
I trusted Paddy when he taught me this, I think Brew doesn't address the PATH issues as well

Reply:
we document to follow Go official install steps